diff --git a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
index 540078a..5274654 100644
--- a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
+++ b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
@@ -482,6 +482,7 @@ exports.LoadUtils = () => {
             ...botOptions,
             ...extraOptions,
         };
+      delete message.__x_id;
 
         // Bot's won't reply if canonicalUrl is set (linking)
         if (botOptions) {
